new/server/models/user.js

```diff
@@ -3,13 +3,29 @@ const mongoose=require('mongoose');
 const Schema=mongoose.Schema;
 
 const userSchema=new Schema({
+    fname:{
+        type:String,
+        required:true
+    },
+    lname:{
+        type:String,
+        required:true
+    },
+    username:{
+        type:String,
+        required:true
+    },
     email:{
         type:String,
         required:true
     },
     password:{
         type:String,
         required:true
+    },
+    profilepic:{
+        type:String,
+        required:true
     }
```

Comment:
Add another field `description` which will serve as his/her description. It should not be included in the form though. 
This will be added when his profile is made and can be edited.

Add `birthdate` which will be a date.

Add `role` which should not be present in the registration page but saves as "user" in the database while registration.

Reply:
where shall i add description field? In form or after registration in any other page like profile page

Reply:
profile page, later. So many fileds not needed here.